packages/playground/src/components/node_selector/TfManualNodeSelector.vue

```diff
@@ -177,7 +177,11 @@ export default {
           case props.filters.ipv4 && farms[0].publicIps.every(p => p.contract_id !== 0):
             throw `Node ${nodeId} is not assigned to a PublicIP`;
           case missingFeatures.length > 0:
-            throw `Node ${nodeId} doesn't support [ ${missingFeatures} ] features`;
+            throw `Node ${nodeId} does not support Feature${missingFeatures.length > 1 ? "s" : ""} ${missingFeatures
+              .slice(0, -1)
+              .join(", ")}${missingFeatures.length > 1 ? " or " : ""}${
+              missingFeatures[missingFeatures.length - 1]
+            }. Please check compatibility or upgrade the node.`;
```

Comment:
looks good 
![image](https://github.com/user-attachments/assets/7cded3bf-b753-40e8-834e-d44c44ef894a)
I'd make it 'wireguard feature' instead of 'Feature wireguard" though

Reply:
just a minor suggestion 
![Screenshot from 2025-02-10 10-44-53](https://github.com/user-attachments/assets/4cade386-9ad6-4945-a3d0-c37c930959fb)

Please add an or before the last feature: "Node 314 does not support IP, Wireguard, **or** Planetary Features. Please check compatibility or upgrade the node."